[#227] Only hiring in CHI, NY, DC, SF

hestherchang opened the pull request:
Added a message to state that we're only hiring in Chicago, NYC, DC, SF to set expecations

hestherchang committed:
[#227] Only hiring in CHI, NY, DC, SF

Added a message to state that we're only hiring in Chicago, NYC, DC, SF to set expecations

fureigh commented on _positions/product-lead.md:
```diff
@@ -8,6 +8,9 @@ active: true
 ###Role Summary
 An 18F Product Lead is responsible for managing the development, delivery, and continuous improvement of products while establishing and managing our relationship with our partners (clients). As a product lead, you’ll work with the 18F intake team to evaluate potential work and communicate to potential clients, partner with the finance and contracts team to close deals, and work with cross-functi ...
 
+At this time, the Product team is only looking for new members who are able to join the team at an office - Chicago, NYC, DC, San Francisco. 
```
A few questions and comments:
- Please change the hyphen to an em-dash (on a Mac, you can type that by pressing `shift-option-hyphen`).
- Let's change `DC` to `D.C.`, per the style guide.
- Do we still technically have a NYC office? If so, let's update https://pages.18f.gov/joining-18f/who-we-are-hiring/ accordingly (in a separate pull request).
- We could parallel the "work out of our offices" phrasing from that page, something like "At this time, we're only hiring Product Leads who are able to work out of our offices in [city list] or [final city]." How does that sound to you?

hestherchang replied:
- Ok - will change hyphen to an em-dash 
- Ok
- checking on this now
- Ok sounds good

fureigh commented:
@hestherchang Thanks for this! I added some questions and comments. You can continue the conversation in that line note or add additional commits to this pull request and assign it back to me when it's ready for review again.

hestherchang committed:
Update product-lead.md

Made edits per Furiegh's suggestion - DC to D.C., wording of statement, and update of city list.

fureigh committed:
Copy tweaks about Product Lead location

fureigh committed:
Remove stray line break

fureigh commented:
@hestherchang, I made a few other copy changes — if they look good to you, please update the Lever text accordingly. Soon I'll [remove the position descriptions from the codebase](https://github.com/18F/joining-18f/issues/232) because they're now hosted on Lever, but I'll merge this in the meantime.
